lib/testing/src/lib/core/pages/data-table-component.page.ts

```diff
@@ -329,6 +329,19 @@ export class DataTableComponentPage {
         BrowserActions.click(resultElement);
     }
 
+    checkRowContentIsDisplayed(content) {
```

Comment:
Can this method be used instead of creating new one: checkContentIsDisplayed?

Reply:
Can't use this method, as there is no columnName displayed in the contentNodeSelectorDialogPage.